### ISSUE TITLE
Can't log in using Databag-Umbrel

### ISSUE THREAD

**Issue by skittle0764** [balzack/databag#86]
I cant log in on Databag on iOS. Im hosting databag using my umbrel server, but it doesnt work. could you also tell me if its possible to log in from the other side of the globe?

**Comment by balzack** [comment]
Yes, you can login from the iOS app to your umbrel hosted databag instance, but with umbrel however you need to use the cloudflare tunnel app. 

The cloudflare setup has a few steps to follow (https://github.com/Radiokot/umbrel-cloudflared/wiki/How-to-set-up-Cloudflare-Tunnel-on-your-Umbrel), and will also require you to purchase a domain. Let me know if you need help with this process.

I have been able to connect to my instance from another country, so I assume the other side of the globe should also work.
